### PR TITLE
Create README.md - gci

### PR DESCRIPTION
**Pull request by anuar2k**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+ItemPipes
```
Comment (TheFlash98) [inline README.md]: Would look better if this is a heading. Just had change it to `# ItemPipes`.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+=====
+**ItemPipes** is a Terasology module that adds item transport system based on pipelines. ItemPipes are useful for automatization purposes - this module lets transport items without the involvement from the player.
```
Comment (TheFlash98) [inline README.md]: I think you mean **automation** and not **automatization**, do fix that.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+Items
```
Comment (TheFlash98) [inline README.md]: Make this a heading as well, preferable `## Items`

File: README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+=====
+
+  - **Pipes** are the base of this module. They are used to transport items. You can connect them together with other pipes to make pipelines and with blocks such as Suction Blocks or chests.
+  - **Suction Block** is used to input items into pipes. Suction Block attracts all items laying on the ground withing 5 blocks long radius to suck them in. If more than one pipe is connected to a Suction Block, it will put the items randomly into one of the connected pipes.
+  
+  _Check out **[AdditionalItemPipes](https://github.com/Terasology/AdditionalItemPipes)** for more interesting items using pipe system!_
+  
+Creating pipelines
```
Comment (TheFlash98) [inline README.md]: Change it to a heading

File: README.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+=====
+To create a pipeline just place two pipes next to each other. They will connect together with each other and make a pipeline. Every pipe is able to have up to 6 connections! If an item comes to a pipe which has more than two connections, it will go further into one of the other connected pipes randomly.
+
+Integration with chests
```
Comment (TheFlash98) [inline README.md]: Make this a heading as well 🙂

**Comment by pollend** [comment]
looks good to me 👍

**Pull request merged**